src/front-door/azext_front_door/custom.py

```diff
@@ -235,6 +235,16 @@ def list_front_doors(cmd, resource_group_name=None):
     return client.list()
 
 
+def list_fd_frontend_endpoints(cmd, resource_group_name, resource_name):
+    client = cf_fd_frontend_endpoints(cmd.cli_ctx, None)
+    return client.list_by_front_door(resource_group_name, resource_name)
+
+
+def get_fd_frontend_endpoints(cmd, resource_group_name, resource_name, item_name):
+    client = cf_fd_frontend_endpoints(cmd.cli_ctx, None)
+    return client.get(resource_group_name, resource_name, item_name)
```

Comment:
It seems these two you should be able to register without needing a custom command. 

```Python
g.command('list', 'list_by_front_door')
g.command('show', 'get')
```

Reply:
The generic command get the front door model, which doesnt have all the information(customHttpsConfiguration) of frontend endpoint. Thats why used the custom command to directly get the frontend endpoint which has all the information.